### PR TITLE
Add a readme for the sample application

### LINKED ISSUE #137: Add Readme for Sample

### PR DESCRIPTION
**Pull request by krisorr**
* Moves the Sample documentation for the sample into a local readme from the Documentation repo.
* Turns Android Pay off in the Manifest, it should be turned on manually if Android Pay is being used.

closes #137

@sav007 please review.

### CODE REVIEW

File: MobileBuy/sample/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+The overflow menu has two items for Customer Accounts, Log In and Orders.
+
+### CustomerLoginActivity
+This activity shows how to sign up a new customer, or log an existing customer in.
+
+### CustomerOrderListActivity
+This activity shows how to retrieve the orders associated with a customer account.
+
+
+## Android Pay
```
Comment (sav007) [inline MobileBuy/sample/README.md]: Do we need to mention anything about android pay env?

Comment (krisorr) [inline MobileBuy/sample/README.md]: what are you thinking of?

Comment (krisorr) [inline MobileBuy/sample/README.md]: ah, I see - Test vs. Production.  Ya, we should probably add some more info on that.

Comment (krisorr) [inline MobileBuy/sample/README.md]: @sav007 - added some more info around Android Pay

**Comment by sav007** [comment]
LGTM

**Pull request merged**